### PR TITLE
Update vagrant documentation to use get.k8s.io for setup.

### PR DESCRIPTION
**Pull request by rjnagal**

### CODE REVIEW

File: docs/getting-started-guides/vagrant.md [hunk 1]
```diff
@@ -5,10 +5,18 @@ Running kubernetes with Vagrant (and VirtualBox) is an easy way to run/test/deve
...
 2. Install latest version of Virtual Box from https://www.virtualbox.org/wiki/Downloads
-4. Get or build a [binary release](binary_release.md)
 
 ### Setup
 
+Setting up a cluster is as simple as running:
+
+```
+export KUBERNETES_PROVIDER=vagrant
+curl -sS https://get.k8s.io | sh
```
Comment (derekwaynecarr) [inline docs/getting-started-guides/vagrant.md]: I think you need to provide two options.

1) you already cloned the repository, so you will just do cluster/kube-up
2) you have no desire to hack on kubernetes and only want to work from a marked release.  [this is what you did]

I think we should not lose 1 as well

Comment (rjnagal) [inline docs/getting-started-guides/vagrant.md]: @derekwaynecarr The idea was to separate out developer guides from getting started stuff. GSG should be user focussed and have minimal distractions. If you like, I can clone up a Vagrant developer guide first.

Comment (derekwaynecarr) [inline docs/getting-started-guides/vagrant.md]: Ah.  That makes sense.  Yes, we need a cloned guide.

Comment (rjnagal) [inline docs/getting-started-guides/vagrant.md]: @derekwaynecarr cloned.

**Comment by derekwaynecarr** [comment]
LGTM

**Pull request merged**